demonstrations/tutorial_how_to_qjit_lgpu.py

```diff
@@ -26,30 +26,33 @@
 to `Catalyst <https://docs.pennylane.ai/projects/catalyst>`__, we suggest to follow the
 `Lightning-GPU <https://docs.pennylane.ai/projects/lightning/en/stable/lightning_gpu/installation.html>`__
 and `Catalyst <https://docs.pennylane.ai/projects/catalyst/en/stable/dev/installation.html>`__
-installation guidelines. On an up-to-date Linux X86_64 or ARM64 CUDA-backed machines,
+installation guidelines. On an up-to-date Linux X86_64 or ARM64 machines with CUDA-12,
```

Comment:
```suggestion
installation guidelines. On Linux X86_64 or ARM64 machines with CUDA-12,
```
No need for `up-to-date` here.